src/tests/SortingProducts.spec.js

```diff
@@ -14,7 +14,7 @@ test.beforeEach(async ({ app }) => {
 test.describe('Saucedemo tests for sorting products on invertory page', () => {
     test('Sorting products by name A-Z', async ({ app }) => {
         const productNamesBeforeSort = await app.inventory.inventoryItems.allTextContents();
-        await app.inventory.sortProductNameAZ();
+        await app.inventory.performProductSort('az');
```

Comment:
Please, create a separate object holding sorting option values. 
It is needed to avoid having 'magic strings'
Use it like `await app.inventory.performProductSort(sortingOptions.AZ);`